### PR TITLE
feat(fibery): Fibery Integration

### LINKED ISSUE #1411: Add integration with Fibery
**Link to service:** https://fibery.io

**Is there a free account option?**: Yes (Beta Invite)

Fibery is next generation work management platform by Targetprocess. Toogl already has Targetprocess integration. Adding integration in all work item and wikis page for Fibery will be a cool addition to Toggl integration list

### PR DESCRIPTION
**Pull request by nmrony**
Add integration to fibery.io, a work management platform by targetprocess

closes #1411

### CODE REVIEW

File: src/scripts/content/fibery.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      const $description =
+        $('.HeaderLayout textarea[name="product-development/name"]', $container) ||
+        $('.HeaderLayout textarea[name="Wiki/name"]', $container);
+      return $description.textContent.trim();
+    };
+
+    const link = togglbutton.createTimerLink({
+      className: 'fibery',
+      description: descriptionSelector
+      // projectName: projectSelector
```
Comment (shantanuraj) [inline src/scripts/content/fibery.js]: I think we can remove this for now

**Comment by shantanuraj** [comment]
@nmrony The popup's styling should be fixed once you rebase the branch on latest master
(fixed in #1425)
So just fixing the `descripitionElement`'s calculation should make this ready to ship

**Comment by nmrony** [comment]
@shantanuraj Awesome I am on it.

File: src/scripts/content/fibery.js (hunk 1 above)
Comment (nmrony) [reply src/scripts/content/fibery.js]: @shantanuraj The comment is removed

**Comment by nmrony** [comment]
@shantanuraj I have pushed fixing the descriptor selector elements. Thanks for your cool suggestions. While applying realised that, `$('.HeaderLayout textarea', $container)` is enough for getting description from all wiki, tutorials, task items, work items. I have rebased `master` into my branch and seems like the popup issue is gone. Adding some snaps for your convenience 
task
wiki
work-item-with-popup

**Comment by shantanuraj** [comment]
@nmrony thanks for your contribution 🙌 
We'll release this in the next update (most probably on Monday)

**Pull request merged**